webui/react/src/components/kit/Breadcrumb.tsx

```diff
@@ -1,25 +1,14 @@
 import { Breadcrumb as AntdBreadcrumb } from 'antd';
+import { ItemType } from 'antd/es/breadcrumb/Breadcrumb';
 import React, { ReactNode } from 'react';
 
 interface BreadcrumbProps {
-  children?: ReactNode;
+  items: ItemType[];
   separator?: ReactNode;
 }
 
-interface BreadcrumbItemProps {
-  children?: ReactNode;
-}
-
-type BreadcrumbItem = React.FC<BreadcrumbItemProps>;
-type BreadcrumbSeparator = React.FC;
-type Breadcrumb = React.FC<BreadcrumbProps> & {
-  Item: BreadcrumbItem;
-  Separator: BreadcrumbSeparator;
-};
+type Breadcrumb = React.FC<BreadcrumbProps>;
```

Comment:
Some props in `Breadcrumb` became deprecated https://ant.design/components/breadcrumb

Reply:
We want to provide a stable API, not blindly reiterate whatever types are in Antd.